### DISCUSSION TITLE
Adult links, region unknown

### DISCUSSION THREAD

**Discussion by BellezaEmporium** [iptv-org/community#259] [Show and tell]
Will be updated over time, viewer discretion is advised.

Full playlist available here : https://xprime.one/ | http://iptv.xprime.one/ (GEOLOCKED to Ukraine)

Few testings I've made : 
XY Max -> Picture (of testing, working on my side): https://imgur.com/a/raCUmd0
Barely Legal TV -> Streamtest : https://streamtest.in/logs/http-iptv-xprime-one-157-mono-m3u8-2dee1bbd-a304-4248-a544-7ab0b36de8f4
PornHD -> Streamtest : https://streamtest.in/logs/http-iptv-xprime-one-r-2-index-m3u8

**Comment by BellezaEmporium** [comment]
@Dum4G I've seen the post before deletion, and well... it's some Xtream-Codes server. Would have been better if it wasn't haha.

**Comment by GBS006** [comment]
great

**Comment by BellezaEmporium** [comment]
I think they found out their channels got noticed.

**Reply by BellezaEmporium** [reply]
![image](https://user-images.githubusercontent.com/30985701/121820673-77ad3c00-cc94-11eb-8fce-065050aaec37.png)

It got geolocked to Ukraine

**Reply by BellezaEmporium** [reply]
https://gofile.io/d/V2tKft

**Reply by BellezaEmporium** [reply]
See if they return a 403 "Forbidden" or a clear geolock message like "Access denied". If it does that, look some info about the host and use a proxy or VPN that goes to the same country.

**Reply by BellezaEmporium** [reply]
That's an XTream-Codes server, they might have changed things : 
![image](https://user-images.githubusercontent.com/30985701/121951873-c70a6f80-cd5b-11eb-9be8-059b79a190ae.png)


Here's the complete m3u file : http://185.71.66.90:8080/get.php?username=net1&password=net1&type=m3u

**Reply by BellezaEmporium** [reply]
Well the problem with Xtream is that it cannot be used for a long time.

**Reply by BellezaEmporium** [reply]
You might take a look at https://github.com/iptv-org/awesome-iptv, in the Providers category. Might be useful.

**Comment by krm28-s** [comment]
Hey bro i need german but pay tv. And turkish can you help me please?

Von meinem iPhone gesendet
